source/lovelace/actions.markdown

```diff
@@ -161,8 +161,11 @@ double_tap_action:
 If you define confirmation as an object instead of boolean, you can add more customization and configurations.
 
 ```yaml
-hold_action:
-  action: toggle
+  - double_tap_action:
```

Comment:
```suggestion
    double_tap_action:
```

Reply:
Why remove the hyphen? It works. Does it work without the hyphen?